.github/workflows/pr.yml

```diff
@@ -74,4 +74,63 @@ jobs:
         with:
           command: deny
           args: check --hide-inclusion-graph license sources advisories
-      
+
+
+  wasm-pack:
+    name: Wasm Build
```

Comment:
Can "Wasm Build" and "Wasm Acceptance tests be combined into one? Then you won't need the "Upload release assets" step

Reply:
I originally had them as one but then split it since they're distinct things (building vs. tests) and thought it might be confusing in a single job. The build succeeds, but the test fails? There was also a separation issue. I wasn't sure about installing the rust toolchain and node in the same job workspace. Though it should work fine, I was separating them. The build job is rust, the test chain is node/npm.

But for the PR it may make sense to have them in one. I mainly wanted the tests as part of the PR but we need to build it first. We don't actually need the release though. What do you think?

Reply:
I agree for a PR having them together makes sense. If for some reason it doesn't build, you would see the error message just the same